src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/useOpprettBehandling.ts

```diff
@@ -103,12 +106,10 @@ const useOpprettBehandling = (
         },
     });
 
-    const migreringsdato = useFelt<FamilieIsoDate>({
-        verdi: '',
-        valideringsfunksjon: (felt: FeltState<FamilieIsoDate>) =>
-            felt.verdi && erIsoStringGyldig(felt.verdi) && erDatoMindreEllerLikMaksdato(felt.verdi)
-                ? ok(felt)
-                : feil(felt, 'Du må velge 01.01.23 eller tidligere som migreringsdato'),
+    const migreringsdato = useFelt<Date | undefined>({
+        verdi: undefined,
+        valideringsfunksjon: (felt: FeltState<Date | undefined>) =>
+            felt.verdi && isValid(felt.verdi) ? ok(felt) : feil(felt, 'Du må velge en gyldig dato'),
```

Comment:
Skal jeg trekke ut dette i en egen valideringsfunksjon som kan brukes for alle dato-felter?

Reply:
Gjerne hvis vi bruker det flere steder allerede

Reply:
✅